spec/models/movement_logic_spec.rb

```diff
@@ -99,4 +99,93 @@
       end
     end
   end
+
+  describe 'Pawn' do
+    it 'white pawns should be able to move up but not down' do
+      game = FactoryBot.create(:game)
+      piece = Piece.create(x: 5, y: 1, type: 'Pawn', game: game, user: game.white_player)
```

Comment:
I believe the y location on this should be 2 instead of 1. This would have it on the bottom line of the board with the queen and king